### PR TITLE
update copyright

### PR DESCRIPTION
**Pull request by NicktheGeek**
@reaktivstudios/reviewers this is ready to review.

### CODE REVIEW

File: languages/en_US.po [hunk 1]
```diff
@@ -3,7 +3,7 @@ msgstr ""
 "Project-Id-Version: gppro-entry-content\n"
 "POT-Creation-Date: 2014-03-22 14:56-0500\n"
 "PO-Revision-Date: 2014-03-22 14:56-0500\n"
-"Last-Translator: Andrew Norcross <andrew@andrewnorcross.com>\n"
+"Last-Translator: Reaktiv Studios <andrew@andrewnorcross.com>\n"
```
Comment (corywebb) [inline languages/en_US.po]: Should we change the email address on this?

**Review by corywebb — CHANGES_REQUESTED** [review]

File: languages/en_US.po (hunk 1 above)
Comment (NicktheGeek) [reply languages/en_US.po]: what do you think it should be? I thought about dev@reaktivstudios.com but I'm not sure that is ideal. Maybe support@?

Comment (corywebb) [reply languages/en_US.po]: @NicktheGeek I discussed this with @jjeaton, and he said just remove the email address.

**Comment by NicktheGeek** [comment]
@corywebb this is updated

**Review by corywebb — APPROVED** [review]
LGTM

**Pull request merged**